bats/fb-verify-packages.bats

```diff
@@ -6,50 +6,12 @@ set -o pipefail
 load os_helper
 load foreman_helper
 
-@test "verify no rh-ruby24, tfm-ror51 or tfm-ror52 SCL packages" {
-  tSetOSVersion
```

Comment:
this is not needed as it's already done in `tIsRHEL`:
https://github.com/theforeman/forklift/blob/fd43a0ad473e618151cb6aabc6e3f5b88406fff8/bats/os_helper.bash#L55-L56

Reply:
more cleanup in https://github.com/theforeman/forklift/pull/1612